packages/user-interface/src/interfaces/paths.ts

```diff
@@ -0,0 +1,14 @@
+export interface Paths {
+  overview: string;
+  cases: string;
+  case: (id?: string | number) => string;
+  tasks: string;
+  task: (id?: string | number) => string;
+  notifications: string;
+  themes: string;
+  forms: string;
+  form: (id?: string | number) => string;
+  account: string;
+  edit_account: string;
+  [key: string]: ((...args: any[]) => string) | string | undefined;
```

Comment:
Is deze nog nodig?

Reply:
Ja die kan handig zijn als je in die interface in je implementatie wilt gebruiken. In ons geval zal er bijvoorbeeld een belastingzaken pad gaan komen in de paths constante, en die zou je zonder die regel niet meer in zijn geheel door kunnen sturen naar het Layout component in de nl-portal-user-interface.